### DISCUSSION TITLE
Small negative power when circuit not in use

### DISCUSSION THREAD

**Discussion by Flat** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#174] [Q&A]
I've just got the hardware for this setup and working on my config. One thing I've noticed that is odd, even after calibrating my CT clamps, when they are either not around a wire, or the circuit is not drawing power, there is a small negative active power reading.

I've seen https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/issues/63 and https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/discussions/59 and I know I can just lambda negatives to 0, but the original discussion is in regards to solar power, and putting excess power back to the grid. I have no solar system and this is entirely on US 120v (split phase). What would be the actual cause of these negative power readings?

**Comment by R0M107** [comment]
Hi,
How much is small negative power ?
Does it also do it when you unplug the plug ?

**Reply by Flat** [reply]
It is around 18watts, it does not show a value when unplugged.
